docs/aad/MeetingConfigurableTab.md

```diff
@@ -0,0 +1,248 @@
+
+#### Step 1: Add context array to application manifest
+
+In your code editor, open the manifest/manifest.template.json file.
+
+Update the version number so it's greater than it was; for example if your manifest was version 1.4, make it 1.4.1 or 1.5.0. This is required in order for you to update the app in Teams.
+
+```json
+"version": "1.5.0"
+```
+
+> NOTE: Make each new version greater than the last so you can update the application in Teams!
+
+Now, immediately under the `scopes` array, in the your configurable tab object, add a new array for defining the `context`:
+
+```json
+"configurableTabs": [
+    {
+      "configurationUrl": "https://<HOST_NAME>/pages/tabConfig.html",
+      "canUpdateConfiguration": true,
+      "scopes": [
+        "teams",
+        "groupchat"
+      ],
+      "context": [
+        "meetingChatTab",
+        "meetingDetailsTab",
+        "meetingSidePanel",
+        "meetingStage"
+      ]
+    }
+  ]
+```
+
+#### Step 2: Rebuild your application package
+
+Open a command line tool in your working folder and execute
+
+```sh
+npm run package
+```
+
+This will generate a new `manifest.json` file and a new application package (northwind.zip).
+
+#### Step 3: Upload the app package
+
+In the Teams web or desktop UI, click "Apps" in the sidebar 1️⃣, then "Manage your apps" 2️⃣. At this point you have three choices:
+
+* Upload a custom app (upload the app for yourself or a specific team or group chat) - this only appears if you have enabled "Upload custom apps" in your setup policy; this was a step in the previous lab
+* Upload an app to your org's app catalog (upload the app for use within your organization) - this only appears if you are a tenant administrator
+* Submit an app to your org (initiate a workflow asking a tenant administrator to install your app) - this appears for everyone
+
+In this case, choose the first option 3️⃣.
+
+![Upload the app](../../assets/03-005-InstallApp-1.png)
+
+Navigate to the Northwind.zip file in your manifest directory and upload it. Although the application is already installed, you are providing a newer version so it will update the application. 
+
+#### Step 4: Run your app
+
+In your working directory execute this command to start the application
+
+```sh
+npm start
+```
+
+### Exercise 2: Add your tab to the pre-meeting stage
+
+#### Step 1: Create Microsoft Teams meeting
+
+Using the Microsoft Teams desktop client, open the `Calendar` app from the sidebar and select `New meeting `in the top right corner.
+
+Enter a `title` and add at least `one required participant`, select `Save` in the top right corner to create the meeting.
+
+#### Step 2: Add tab
+
+Select the meeting that you just created in the Calendar app, in the flyout, select `Edit` to open the meeting details.
+
+Select the `(+)` Add tab button in the meeting and select the `Northwind Orders` app, in the tab configuration page, select a `Category` of products you want to display in the tab.
```

Comment:
From @rabwill 

If I may suggest pls add 'search for northwind app and select' in Step 2: Add tab

Reply:
Good suggestion.